App/Application.cpp

```diff
@@ -837,11 +846,9 @@ void
 Application::onPanSpectrumStop()
 {
   if (m_scanner != nullptr) {
-    delete m_scanner;
-    m_scanner = nullptr;
+    m_scanner->stop();
```

Comment:
This is perhaps what is causing the errors to be shown after clicking "stop".

Reply:
I think so too. I had to defer the deletion of the scanner because the waterfall holds a reference to one of its spectrum views, and uses that data when you zoom into the waterfall. Perhaps there's some issue with synchronization of the stop within suscan. I still need to try this out on Linux to see if I get the same errors as you; I hadn't seen any errors on Mac.